src/cfg.rs

```diff
@@ -67,13 +71,15 @@ impl Default for KatexConfig {
             macros: None,
             block_delimiter: Delimiter::same("$$".into()),
             inline_delimiter: Delimiter::same("$".into()),
+            pre_render: cfg!(feature = "pre-render"),
```

Comment:
Nice trick. But, would be better to enforce consistent behaviors (this might come back and bite us).

Reply:
I don't know how to enforce consistent behaviors. I think enabling the pre-render feature by default is enough.